fe/fe-core/src/main/java/com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java

```diff
@@ -138,9 +140,8 @@ public Void visitCreateMaterializedViewStatement(CreateMaterializedViewStatement
             }
             tableNameTableMap.forEach((tableNameInfo, table) -> {
                 if (table == null) {
-                    throw new SemanticException("Materialized view do not support table: " + tableNameInfo.getTbl() +
-                            " do not exist in database: " + tableNameInfo.getCatalog() + ":" +
-                            tableNameInfo.getDb());
+                    throw new SemanticException("Referenced table %s.%s.%s not exist", tableNameInfo.getCatalog(),
```

Comment:
Is the catalog being used right now ?

Reply:
This table shoud not be null, otherwise it will report error at analyze stage for mv's select stmt